Makefile

```diff
@@ -11,6 +14,24 @@ lintCheck:
 	yamllint -c .yamllint.yaml telco-core/install/
 	yamllint -c .yamllint.yaml telco-hub/
 
+# markdownlint rules, following: https://github.com/openshift/enhancements/blob/master/Makefile
+.PHONY: markdownlint-image
+markdownlint-image:  ## Build local container markdownlint-image
+	$(CONTAINER_TOOL) image build -f ./hack/Dockerfile.markdownlint --tag $(IMAGE_NAME)-markdownlint:latest ./hack
```

Comment:
Instead of building the container image through the Makefile target, can we install markdownlint into the CI container directly and use it from there? We build the CI container here:
https://github.com/openshift/release/blob/master/ci-operator/config/openshift-kni/telco-reference/openshift-kni-telco-reference-main.yaml#L3-L7
Note that we've had discussions about moving the dockerfile.literal from there into a Dockerfile maintained here (eg hack/Dockerfile.ci) but haven't yet done so.

Reply:
Yes, integration with openshift/release was the next step once this got merged, I should have made this comment in the PR description.
With this change though, everyone can also run the tests locally.

Reply:
When you update the openshift release will you remove the image build here? ie the image should be built through prow and not through the markdownlint: target. Then the markdownlint: target can just call markdownlint.sh directly and not through another container.

Reply:
The image built here is purely for running markdownlint locally.
In openshift-release we'd have the following under the `ci-operator/config/...`:
```yaml
test_binary_build_commands: |
  ./hack/install-markdownlint.sh

tests:
...
- always_run: false
  as: markdownlint
  commands: |
    hack/markdownlint.sh
  container:
    from: test-bin
```
Reference to the ci-operator docs about these [here](https://docs.ci.openshift.org/docs/internals/#build-graph-traversal).
I believe this addresses your concerns. Kindly confirm.